shifterator/helper.py

```diff
@@ -115,11 +115,14 @@ def get_missing_scores(type2score_1, type2score_2):
     type2score_1, type2score_2: dict
         Keys are types and values are scores
 
-    Output
+    Returns
     ------
```

Comment:
Just add one more "-" so it goes all the way under "Returns". Think it might mess up automated documentation otherwise